### PR TITLE
Add UniswapV2 support

### PR DESCRIPTION
**Pull request by NickCuso**

### CODE REVIEW

**Comment by NickCuso** [comment]
hmm this was passing locally but now failing.. investigating..

**Comment by NickCuso** [comment]
Running locally it seems to pass approx 1 out of every 3 runs. hmm

**Comment by NickCuso** [comment]
Okay it seems to be passing consistently now.

**Comment by NickCuso** [comment]
hmm CI failed again. not sure why the test is not reliable..

File: src/protocols/uniswapV2.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (julien51) [inline src/protocols/uniswapV2.json:1]: I would assume you have a way to ensure this is the ABI coming from them?

File: src/protocols/uniswapV2.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+const { truffleContract } = require("../helpers");
+const uniswapJson = require("./uniswapV2.json");
+
+module.exports = {
+  mainnetFactoryAddress: "0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f",
```
Comment (julien51) [inline src/protocols/uniswapV2.js:5]: Verified both contracts are indeed correct

File: src/protocols/uniswapV2.json (hunk 1 above)
Comment (NickCuso) [reply src/protocols/uniswapV2.json:1]: I took it from the etherscan page

**Pull request merged**